windows/src/developer/TIKE/compile/CompileKeymanWeb.pas

```diff
@@ -1625,6 +1649,27 @@ function TCompileKeymanWeb.ValidateLayoutFile(var sLayoutFile: string; const sVK
         if FDictionary.IndexOf(FId) < 0 then
           ReportError(0, CWARN_TouchLayoutCustomKeyNotDefined, 'Key "'+FId+'" on layer "'+FLayer.Id+'", platform "'+FPlatform.Name+'", is a custom key but has no corresponding rule in the source.');
       end;
+
+      //
+      // Check that if the key has a *special* label, it is available in the target version
+      //
+      if FText.StartsWith('*') and FText.EndsWith('*') and (FText.Length > 2) then
+      begin
+        // We do not support '*special*' labels on non-special keys in Keyman 13
+        // or earlier, except for ZWNJ, which will be transformed in function
+        // TransformSpecialKeys14 to '<|>', and  which does not require the custom
+        // OSK font.
```

Comment:
Just a suggestion, as this version of the comment feels clearer to me.  Feel free to disregard it.

```suggestion
        // Keyman versions before 14 do not support '*special*' labels on non-special keys.
        // ZWNJ use, however, is safe because it will be transformed in function
        // TransformSpecialKeys14 to '<|>',  which does not require the custom OSK font.
```